src/main/java/tutoraid/model/student/Student.java

```diff
@@ -180,9 +180,9 @@ public String toString() {
 
         builder.append("\nProgress: ")
                 .append(getLatestProgress())
-                .append("; Payment Status: ")
-                .append("; Lessons: ")
-                .append(getLessons());
+                .append("\nLessons: ")
+                .append(getLessons())
+                .append("\n");
```

Comment:
Purpose of adding a newline at the end is to improve formatting in the feedback message for `add -sl` and `-del -sl`